docs/Installation.md

```diff
@@ -38,18 +38,6 @@ This instruction covers the following operating systems:
 
 ### 1. CentOS
 
-Install binary dependencies:
-
-```sh
-sudo yum install perl-JSON-XS
```

Comment:
When libraries are installed from binaries it will be more consistent what you get. It also helps for future packaging. I think it should be kept as-is.